haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
+    """Create an initial admin user. Only valid for the database auth backend.
+
+    This must be run on the HaaS API server, with access to haas.cfg and the
+    database. It will create an initial user named <username> with password
+    <password>, who will have administrator priviledges. This user can then
+    create additional users via the API.
+    """
+    if not config.cfg.has_option('extensions', 'haas.ext.auth.database'):
+        sys.exit("'make_inital_admin' is only valid with the database auth backend.")
+    from haas import model
+    from haas.ext.auth.database import User
+    model.init_db(create=False)
+    db = model.Session()
+    db.add(User(label=username, password=password, is_admin=True))
+    db.commit()
+
```

Comment:
Perhaps make_initial_admin should be somewhere other than the haas CLI---many openstack services have a split between client library and -admin tool.  Since we're moving towards splitting the CLI out, we should keep that in mind.